File: a003.py
# https://replit.com/@andy8787main/a003#main.py
# https://zerojudge.tw/ShowProblem?problemid=a003
 
m,d = [int(i) for i in input().split(' ')] #分割字串轉數字
s = (m*2+d)%3
if (s == 0):
    print("普通")
elif (s == 1) :
    print("吉")
else:
    print("大吉")

File: a003.cpp
// https://replit.com/@andy8787main/a003-cpp#main.cpp
// https://zerojudge.tw/ShowProblem?problemid=a003
#include <iostream>
using namespace std;

int main() {
    int M, D, S;
    cin >> M >> D;
    S = (M*2+D)%3;
    cout << ((S == 0) ? "普通" : ( (S == 1) ? "吉" : "大吉"));
} 
